activejob/lib/active_job/queue_name.rb

```diff
@@ -29,22 +29,35 @@ def queue_name_from_part(part_name) #:nodoc:
         name_parts = [queue_name_prefix.presence, queue_name]
         name_parts.compact.join(queue_name_delimiter)
       end
+
+      def queue_name=(v)
+        @queue_name = v
+      end
+
+      def queue_name
+        name = defined?(@queue_name) ? instance_variable_get(:@queue_name) : default_queue_name
```

Comment:
oops), i will remove `instance_variable_get`

but without `defined?` i see:
`warning: instance variable @queue_name not initialized`